portfolio/src/main/webapp/script.js

```diff
@@ -38,14 +41,32 @@ async function getComments(commentLimit) {
 }
 
 function filterComments(value) {
-  var commentLimit = parseInt(value);
-  if (commentLimit >= 0 ) {
+  const commentLimit = Number(value);
+  if (commentLimit >= 0) {
     getComments(commentLimit.toString(10));
   }
 }
 
+async function deleteComments() {
+  const response = await fetch('/delete-data', {
+    method: 'POST'
+    })
```

Comment:
nit: indentation seems wrong. I recommend adding an auto formatting plugin to your editor.